cobigen-templates/templates-devon4j/src/main/templates/crud_openapi_angular_client_app/templates/src/app/model/${variables.etoName#lower_case}.ts.ftl

```diff
@@ -7,7 +7,11 @@
  *
  * NOTE: This class is auto generated by CobiGen.
  */
-<#include "model.ts.ftl">
+<#compress>
+<#list model.properties as property>
```

Comment:
I think you forgot to pull the latest changes from CobiGen master. This file is outdated, it should not be modified.